src/number_theory/padics/padic_integers.lean

```diff
@@ -360,11 +347,10 @@ lemma mul_inv : ∀ {z : ℤ_[p]}, ∥z∥ = 1 → z * z.inv = 1
     rw [norm_eq_padic_norm] at h,
     rw dif_pos h,
     apply subtype.ext_iff_val.2,
-    simp [mul_inv_cancel hk],
+    simp [mul_inv_cancel hk]
```

Comment:
Once again we don't have an explicit rule here, but I'd like to see one.

Reply:
Note that in the absense of a rule, it's best not to change any code that you're not making significant changes to already.